src/virtual_elements.js

```diff
@@ -118,7 +118,7 @@ var hasChangedAttrs = function(unused1, unused2, unused3, var_args) {
   }
 
   if (attrsChanged) {
```

Comment:
I suppose the if check can be removed as well because if the attributes didn't change, i would be > arguments.length and we would just skip over the loop.